app.py

```diff
@@ -1,27 +1,34 @@
-from flask import Flask, redirect, render_template
+from flask import Flask, redirect, render_template, session, request
 
 from src.repositories.movie_repository import get_movie_repository
 
 app = Flask(__name__)
 
+
+
+
+
 # Get the movie repository singleton to use throughout the application
 movie_repository = get_movie_repository()
 
-
 @app.get('/')
 def index():
     return render_template('index.html')
 
-
+#Jaidens List All Movies Function
 @app.get('/movies')
 def list_all_movies():
-    # TODO: Feature 1
-    return render_template('list_all_movies.html', list_movies_active=True)
+    movies = []
+    moviesList = movie_repository.get_all_movies()
+    for i in moviesList:
+        movie = movie_repository.get_movie_by_id(i)
+        movies.append(movie)
```

Comment:
You may not need to append the movie...I'm not too sure until everything comes together, but when the movies are created it's already appended to the movies.